README.md

```diff
@@ -102,13 +98,23 @@ yarn ts
 
 2. Create a new branch, and make sure to push it. This is where package version updates, and git tags will be pushed from.
 
-3. Publish all packages to NPM using Lerna. In the future, this will also include automatic changelog creation.
+3. Generate a changeset for each package.
+
+```
+yarn changeset bump
```

Comment:
Need to also commit these changes too right?

When do we commit?

Reply:
Updated!